src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -4,6 +4,7 @@ import * as React from 'react';
 import {FlatList, StyleSheet, Text, View} from 'react-native';
 
 import PropTypes from 'prop-types';
+import Accordion from 'react-native-collapsible/Accordion';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Accordion` from `react-native-collapsible/Accordion` is a new addition. Ensure that the package `react-native-collapsible` is included in your project dependencies.
```diff
+ import Accordion from 'react-native-collapsible/Accordion';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->